mod-di-converter-storage-server/src/main/java/org/folio/rest/impl/DataImportProfilesImpl.java

```diff
@@ -88,6 +87,9 @@ public class DataImportProfilesImpl implements DataImportProfiles {
     "Please ensure your Action and Mapping profiles are of like types and try again.";
   private static final String INVALID_ACTION_PROFILE_ACTION_TYPE = "Can't create ActionProfile for MARC Bib record type with Create action";
   private static final String INVALID_ACTION_PROFILE_LINKED_TO_JOB_PROFILE = "ActionProfile with id '%s' and action UPDATE requires linked MatchProfile";
+  public static final String LINKED_ACTION_PROFILES_WERE_NOT_FOUND = "Linked ActionProfiles with ids %s were not found";
```

Comment:
Do these fields need to be public?

Reply:
No, these fields should not be public. Fixed, that. Thank you